src/cpp/session/modules/SessionPackages.R

```diff
@@ -1159,3 +1171,95 @@ if (identical(as.character(Sys.info()["sysname"]), "Darwin") &&
    # delegate to 'setInternet2'
    utils::setInternet2(value)
 })
+
+.rs.addFunction("parseSecondaryReposIni", function(conf) {
+   lines <- readLines(conf)
+   repos <- list()
+
+   for (line in lines) {
+     part <- strsplit(line, "=")[[1]]
```

Comment:
I think we should support comments here; by convention we often include a sample config file that contains comments with a brief description of the file's purpose and some usage example.

Reply:
Yeah, we can use the boost parser I introduced for parsing `repos.conf` in the C++ codebase, that one does support comments.